assets/css/bundle.css

```diff
@@ -356,6 +356,16 @@ border-bottom-color: #14598a !important;
 .al-page-blog-post .container{
   padding: 3rem 15px !important;
 }
+@media (max-width: 800px){
+  .img-fluid, .img-thumbnail {
```

Comment:
I'm not sure if these will cause unexpected changes elsewhere in the site.

Reply:
Upon further review, that rule isn't even necessary.  The img-thumbnail class is not used anywhere in the blog section, and the img-fluid class has that behavior by default.  

This latest commit removes them entirely.  It also adds the ending curly brace in the blog.css file I missed.